### PR TITLE
[NCL-3812] Improve Build Configuration section on Project detail page

### PR DESCRIPTION
**Pull request by jomrazek**
### Checklist:

* [ ] Have you added a note in the [CHANGELOG wiki](https://github.com/project-ncl/pnc/wiki/Changelog) for your change if user-facing?
* [ ] Have you added unit tests for your change?

### CODE REVIEW

**Comment by jomrazek** [comment]
@matedo1 Can you look at it if I'm missing something please? Looks like it's working when I'm testing it

File: ui/app/projects/detail/pnc-project-detail-page.html [hunk 1]
```diff
@@ -96,9 +96,5 @@
 </pnc-header>
 
 <div class="row">
-  <pnc-toolbar filtering-page="$ctrl.buildConfigsFilteringPage" filtering-fields="$ctrl.buildConfigsFilteringFields"></pnc-toolbar>
-</div>
-<div>
-  <pnc-build-configs-list build-configs="$ctrl.buildConfigsFilteringPage.data" display-fields="$ctrl.buildConfigsDisplayFields"></pnc-build-configs-list>
-  <pnc-page-controls class="pull-right m-t-10 m-b-10" pnc-page="$ctrl.buildConfigsFilteringPage" ng-if="$ctrl.buildConfigsFilteringPage"></pnc-page-controls>
+  <pnc-build-configs-data-table page="$ctrl.buildConfigs" on-remove="null" refresh-build-configs="$ctrl.refreshBuildConfigs"></pnc-build-configs-data-table>
```
[the remark this answers is not in the document]
Comment (matedo1) [reply ui/app/projects/detail/pnc-project-detail-page.html:99]: in this case I would also prefer to remove `refresh-build-configs="$ctrl.refreshBuildConfigs"` part as this part of the code should not be used in the future

File: ui/app/projects/detail/pncProjectDetailPage.js [hunk 2]
```diff
@@ -83,6 +80,9 @@
       $ctrl.project.$update().finally(reload);
     }
 
+    function refreshBuildConfigs(buildConfigs) {
```
Comment (matedo1) [inline ui/app/projects/detail/pncProjectDetailPage.js:83]: the same as above, I would prefer to remove this method

**Review by matedo1 — CHANGES_REQUESTED** [review]
Thanks for raising this fix! There are just minor comments, otherwise it looks good!

**Review by matedo1 — APPROVED** [review]

**Pull request merged**